crates/next-dev/src/main.rs

```diff
@@ -101,32 +102,24 @@ fn default_host() -> IpAddr {
     IpAddr::V4(std::net::Ipv4Addr::new(0, 0, 0, 0))
 }
 
-#[cfg(not(feature = "cli"))]
-fn main() -> Result<()> {
-    unimplemented!("Cannot run binary without CLI feature enabled");
-}
-
-#[tokio::main]
-#[cfg(feature = "cli")]
-async fn main() -> Result<()> {
+pub async fn start_server(options: &DevServerOptions) -> Result<()> {
```

Comment:
I think this should be moved to `lib.rs`, `main.rs` should be just for the binary build, nothing else

Reply:
yeah good point.